cypress/component/annot-missing-values.cy.js

```diff
@@ -28,13 +28,30 @@ describe("missing values", () => {
             }
         );
 
-    it.only('handles lack of description gracefully', () => {
-            cy.mount(annotMissingValues, {
-                    propsData: props,
-                    computed: Object.assign(getters, {valueDescription: () => (col, mis) => null })
-                }
-            );
-        }
-    );
+        it('handles lack of description gracefully', () => {
```

Comment:
Do we want to standardize the field(s) we're using for mock store fields and functions? I'm referring specifically to how getters is defined at the top of this file as a separate field.

Reply:
Yeah. Let's just pick something that mimics what we're injecting into the mount.

`getters`, `actions` (are we doing this?), `mutations`, `props`

Reply:
Agreed. And I think `state` is a good idea for passing just the store-state object. Should we put that in the wiki?

Did you want to change something about the tests here before merge too?

Reply:
Agreed. And I think `state` is good Should we put that in the wiki?
Also: anything you'd like to change here in the test to match that?

Reply:
Ah, I see. If we do wrap it in a big `store` object, we'll need to reference them when passing to the `mount()` directive though, yes? What do you think is the benefit of either you think?

Reply:
Just the mimicry of the actual app internals. For instance, `props` would not be in the `store` object. But that's about it.

Reply:
ah. OK. that makes sense to me!